pennylane/__init__.py

```diff
@@ -133,6 +136,97 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, format: str):
+    """load(quantum_circuit_object, format)
```

Comment:
I haven't seen this pattern before. What does it do when you put the signature right after the triple quotes?

Reply:
I got it from the `device` functions, I would assume it is used for the docs. @josh146 ?

Reply:
If you want the object signature to be displayed in the docs differently to the actual docstring, you can 'overwrite' Sphinx by putting your desired displayed signature as the first line of the docstring.

We use this in a couple of places in PennyLane, mostly the operations, to avoid the user seeing certain defaults that are there for programmatic reasons (i.e., `wires=None`).

Here, though, there is no need for it, since it matches the actual docstring.